commons/src/main/java/io/aiven/kafka/tieredstorage/commons/manifest/index/AbstractChunkIndex.java

```diff
@@ -91,7 +91,7 @@ public Chunk findChunkForOriginalOffset(final int offset) {
             //                ^
             //             offset
             final int firstOffsetBeyondCurOriginalChunk = (chunkI + 1) * originalChunkSize;
-            if (offset < firstOffsetBeyondCurOriginalChunk) {
+            if (offset <= firstOffsetBeyondCurOriginalChunk) {
```

Comment:
To warn in advance: this seems an incorrect change. It breaks the `VariableSizeChunkIndexBuilderTest` and `VariableSizeChunkIndexBuilderTest` tests and it's a legit failure.

Reply:
Yep, fixed